guess.cpp

```diff
@@ -0,0 +1,63 @@
+#include<iostream>
+#include<fstream>
+/*Vick Ye
+ *1h 51m 41s
+ *10/10
+ *strings, organizing animals and characteristics, nested loops
+ */
+//array representing potential guesses, with inner array representing characteristics
+std::string A[100][100];
+int characteristics[100];
```

Comment:
It is the number of characteristics, not characteristics. If you put `num` in the name, then we always know it is an integer.